### PR TITLE
UPDATED HOMEPAGE

### PR DESCRIPTION
**Pull request by jaiyankargupta**
Updated Homepage but some issues in footer #2

### CODE REVIEW

File: client/src/App.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/* index.css */
+@import 'tailwindcss/base';
+@import 'tailwindcss/components';
+@import 'tailwindcss/utilities';
+
+/* Your custom styles */
```
Comment (Himasnhu-AT) [inline client/src/App.css:6]: remove comments

File: client/src/App.jsx [hunk 2]
```diff
@@ -41,17 +43,20 @@ const attractionData = {
...
-      <DestinationInput />
-      {/* Pass the actual data or variable to the SearchResults component */}
-      {/* <SearchResults results={searchResults} /> */}
-      {/* Similarly, for other components */}
-      <RouteCard routes={routeData} />
-      <AccommodationCard accommodation={accommodationData} />
-      <DiningOptionCard diningOption={diningOptionData} />
-      <AttractionCard attraction={attractionData} />
-    </div>
+    // <div className=' bg-white text-black flex justify-center items-center'>
```
Comment (Himasnhu-AT) [inline client/src/App.jsx:46]: remove extra comments

File: client/src/components/Header.jsx
Comment (Himasnhu-AT) [inline client/src/components/Header.jsx:1]: remove un-necessary comments

File: client/src/components/HomePage.jsx
Comment (Himasnhu-AT) [inline client/src/components/HomePage.jsx:1]: remove unnecessary comments

File: client/src/components/footer.jsx
Comment (Himasnhu-AT) [inline client/src/components/footer.jsx:1]: add internal page links, contributors so on....

File: client/src/components/travelGuideLogo.jpeg
Comment (Himasnhu-AT) [inline client/src/components/travelGuideLogo.jpeg:1]: move this file to @/public folder

File: client/src/components/travelGuideLogo.png
Comment (Himasnhu-AT) [inline client/src/components/travelGuideLogo.png:1]: move this file to @/public folder

**Review by Himasnhu-AT — CHANGES_REQUESTED** [review]
@jaiyankargupta 
resolve the comments:
- removing unnecessary comments,
- make images in /public folder
- update footer component

File: client/src/components/HomePage.jsx
Comment (jaiyankargupta) [reply client/src/components/HomePage.jsx:1]: okay

File: client/src/App.css (hunk 1 above)
Comment (jaiyankargupta) [reply client/src/App.css:6]: DONE

File: client/src/App.jsx (hunk 2 above)
Comment (jaiyankargupta) [reply client/src/App.jsx:46]: DONR

File: client/src/components/Header.jsx
Comment (jaiyankargupta) [reply client/src/components/Header.jsx:1]: DONE

**Review by Himasnhu-AT — APPROVED** [review]

**Pull request merged**